lib/omnisharp-atom/features/code-action.ts

```diff
@@ -84,6 +85,25 @@ class CodeAction implements OmniSharp.IFeature {
         }));
     }
 
+    private getRequest(client: OmniSharp.ExtendApi, codeAction?: number) {
+        var editor = atom.workspace.getActiveTextEditor();
+        var range = editor.getSelectedBufferRange();
+
+        var request = client.makeDataRequest<OmniSharp.Models.CodeActionRequest>({
+            WantsTextChanges: true,
+            SelectionStartLine: range.start.row + 1,
```

Comment:
Checked, Looked at the request that comes in and unless I add the `+1` it comes in zero based.

So I'm a little confused